CHANGELOG.rst

```diff
@@ -15,11 +15,32 @@ Breaking changes:
 Improvements and new features:
 
  - Handles spaces in ``gcov`` path. (:issue:`385`)
+ - Early fail when output cannot be created. (:issue:`346`)
+ - Add option :option:`--txt` for text output. (:issue:`387`)
+ - Add option :option:`--csv` for CSV output. (:issue:`376`)
+ - Add :option:`--exclude-lines-by-pattern` to filter out source lines by arbitrary
+   regex. (:issue:`356`)
+ - Compare paths case insensitive if file system of working directory is case insensitive. (:issue:`329`)
+ - Only output 100.0% in text and HTML output if really 100.0%, else use 99.9%. (:issue:`375`)
+ - Changes for HTML output format:
+
+   - Change legend for threshold in html report. (:issue:`369`)
+   - Use HTML title also for report heading. Default value for :option:`--title` changed. (:issue:`269`)
```

Comment:
The documentation build helpfully fails because no `--title` option exists. Issue ID swapped some numbers.
```suggestion
   - Use HTML title also for report heading. Default value for :option:`--html-title` changed. (:issue:`296`)
```

Reply:
I'll check this.